### PR TITLE
Add Rename option in controller context menu

### PR DESCRIPTION
**Pull request by DeRobyJ**
This time I did it with branches, so I hope everything is fine!

(With this, I'm fixing #2744 )

### CODE REVIEW

**Comment by Umcaruje** [comment]
Please convert all your indentation from spaces to tabs.

**Comment by DeRobyJ** [comment]
I can just update it in my branch and this pull request will be fixed for that?

**Comment by Umcaruje** [comment]
@DeRobyJ Yup, when you push to your branch it automatically gets updated in the PR.

**Comment by DeRobyJ** [comment]
Fixed it.
Problem was, Qt creator was set up to put spaces instead of tabs °-°

File: src/gui/widgets/ControllerView.cpp [hunk 1]
```diff
@@ -173,16 +176,16 @@ void ControllerView::contextMenuEvent( QContextMenuEvent * )
 {
 	QPointer<CaptionMenu> contextMenu = new CaptionMenu( model()->displayName(), this );
 	contextMenu->addAction( embed::getIconPixmap( "cancel" ),
-						tr( "&Remove this plugin" ),
+						tr( "&Remove this controller" ),
 						this, SLOT( deleteController() ) );
+	contextMenu->addAction( tr("Re&name this controller"), this, SLOT( renameController() ));
```
Comment (Umcaruje) [inline src/gui/widgets/ControllerView.cpp]: Is this a typo? should it be `&Rename this controller`?

Comment (DeRobyJ) [inline src/gui/widgets/ControllerView.cpp]: Nope, the &r is used for Remove already

**Comment by Umcaruje** [comment]
Tested this out, works good, looks like a valuable addition and the code lgtm. Merging.

**Pull request merged**